src/gui/advancedsettings.cpp

```diff
@@ -537,6 +540,9 @@ void AdvancedSettings::loadAdvancedSettings()
             + ' ' + makeLink("https://www.libtorrent.org/reference-Settings.html#validate_https_trackers", "(?)"))
             , &m_checkBoxValidateHTTPSTrackerCertificate);
 #endif
+    // Block peer connections on privileged ports
+    m_checkBoxBlockPeersOnPrivilegedPorts.setChecked(session->blockPeersOnPrivilegedPorts());
+    addRow(BLOCK_PEERS_ON_PRIVILEGED_PORTS, (tr("Block peer connections on privileged ports") + ' ' + makeLink("https://libtorrent.org/single-page-ref.html#no_connect_privileged_ports", "(?)")), &m_checkBoxBlockPeersOnPrivilegedPorts);
```

Comment:
I would suggest: "Disallow connections to peers on privileged port".

Reply:
I prefer Nyriox's suggestion, but others are free to chime in.

Reply:
I'll explain myself a bit more.
Personally I think "block" means some thing is initiated but being cut off in the middle, while "disallow" means it is not permitted in the first place.

Second, IMO "Block peer connections on privileged ports" would sound similar to "don't connect to peers from our local privileged ports", of course that is nonsense but I would still like to clear that ambiguity.

Reply:
"Block peer connections on privileged ports" also sounds like incoming peer connections using their privileged ports trying to connect to your listening port that's not a privileged port.
TCP connections are likely to use ephemeral ports, but uTP may very well use the same port as the listening port...so this issue is a real.

The elephant-in-the-room is the purpose of this feature in the first place -- to reduce DDoS-Attack potential of qBitTorrent, particularly if fed "target" ip + port addresses by malicious peers or trackers.

Reply:
Do you suggest a different name for the functions/variables as well?

Reply:
>IMO "Block peer connections on privileged ports" would sound similar to "don't connect to peers from our local privileged ports", of course that is nonsense but I would still like to clear that ambiguity.

Block peers that use privileged port.
Block peer connections from privileged ports.
>Personally I think "block" means some thing is initiated but being cut off in the middle, while "disallow" means it is not permitted in the first place.

"Blocked" is currently used when peer is disallowed by IP filter.

Reply:
>Do you suggest a different name for the functions/variables as well?

No, they are fine.

Reply:
Alright, adapted to Chocobo's suggestion.